src/transaction/sign.ts

```diff
@@ -5,10 +5,11 @@ import keypairs from 'ripple-keypairs'
 
 import type { Client, Wallet } from '..'
 import { ValidationError } from '../common/errors'
+import { Transaction } from '../models/transactions'
 import { xrpToDrops } from '../utils'
 import { computeBinaryTransactionHash } from '../utils/hashes'
 
-import { SignOptions, KeyPair, TransactionJSON } from './types'
+import { SignOptions, KeyPair } from './types'
```

Comment:
Are we going to move types? or will that wait for the "Rip out `src/transaction` and `src/ledger`" PRs

Reply:
We have "rip out `src/transaction/sign`" and "rip out all the other old `src` stuff" tickets lined up - these types won't be needed once we do the former